applications/API/endpoints.py

```
@@ -0,0 +1,538 @@
+from django_filters.rest_framework import DjangoFilterBackend
+from drf_spectacular.utils import (
+    OpenApiExample,
+    OpenApiParameter,
+    OpenApiResponse,
+    OpenApiTypes,
+    extend_schema,
+    extend_schema_view,
+)
+from loguru import logger
+from rest_framework import status
+from rest_framework.authtoken.models import Token
```

Pylint (3.3.7)

[warning] Unused Token imported from rest_framework.authtoken.models

(W0611)